CONTRIBUTING.md

```diff
@@ -1,6 +1,24 @@
 # Contributing
 
+## Adding plugins
+
+When adding or upgrading a plugin, we should see changes in both `package.json`
+and `yarn.lock`.
+
+You'll need to [install yarn](https://yarnpkg.com/en/docs/install).
```

Comment:
oh, I might see my mistake -- is it running those 3 docker commands in 3 separate containers or something like that? 😞 

I did it on multiple lines because there are multiple files we expect to change as part of these commands...